listenbrainz/db/feedback.py

```diff
@@ -67,20 +80,22 @@ def delete(feedback: Feedback):
         Args:
             feedback: An object of class Feedback
     """
-    conditions = ["user_id = :user_id"]
-    args = {"user_id": feedback.user_id}
+    params = {"user_id": feedback.user_id}
 
-    if feedback.recording_msid:
-        conditions.append("recording_msid = :recording_msid")
-        args["recording_msid"] = feedback.recording_msid
-
-    if feedback.recording_mbid:
-        conditions.append("recording_mbid = :recording_mbid")
-        args["recording_mbid"] = feedback.recording_mbid
+    if feedback.recording_msid is not None and feedback.recording_mbid is not None:
```

Comment:
I see we're explicitly checking against None here, is there a case when they might be `""`? (same question for the insert method)

In fact, I see that the column type for these fields is UUID, is there a case where we might insert invalid data and get an exception? I see that the `Feedback` data model has a validator for these fields which we use in the API endpoint, so I think we're fine here right?

Reply:
As discussed will fix in separate PR and also add validator for at least one of mbid and msid to be present.